flytekit/interaction/click_types.py

```diff
@@ -57,8 +57,11 @@ def convert(
         self, value: typing.Any, param: typing.Optional[click.Parameter], ctx: typing.Optional[click.Context]
     ) -> typing.Any:
         p = pathlib.Path(value)
+        # set remote_directory to false if running pyflyte run locally. This makes sure that the original
+        # directory is used and not a random one.
+        remote_directory = None if getattr(ctx.obj, "is_remote", None) else False
```

Comment:
i just dont like this false thing, but what can you do.
I wanted to add FlyteFile.for_remote() or something like that